src/routes/safe/components/Apps/types.d.ts

```diff
@@ -1,15 +1,20 @@
+export enum SAFE_APP_FETCH_STATUS {
+  LOADING = 'LOADING',
+  SUCCESS = 'SUCCESS',
+  ERROR = 'ERROR',
+}
+
 export type SafeApp = {
   id: string
   url: string
   name: string
   iconUrl: string
   disabled?: boolean
-  isDeletable?: boolean
```

Comment:
Is this something temporal? Or delete functionality is not going to be implemented?

Reply:
It's calculated when the app is loaded, that's why I removed the attribute. 
https://github.com/gnosis/safe-react/pull/1600/files#diff-b6ca52868702d0ce84b5ac4a529ffe7ea2c6c7c7bb6eb476d1bfbb2d0651b1f6R192